test/statsd/ecs_resources/ec2_launch/daemon/ecs_taskdef.tpl

```diff
@@ -0,0 +1,90 @@
+[
+  {
+    "name": "cloudwatch_agent",
+    "image": "${cwagent_image}",
+    "essential": true,
+    "secrets": [
+      {
+        "name": "CW_CONFIG_CONTENT",
+        "valueFrom": "${cwagent_ssm_parameter_arn}"
+      },
+      {
+        "name": "PROMETHEUS_CONFIG_CONTENT",
+        "valueFrom": "${prometheus_ssm_parameter_arn}"
+      }
+    ],
+    "logConfiguration": {
+      "logDriver": "awslogs",
+      "options": {
+        "awslogs-region": "${region}",
+        "awslogs-stream-prefix": "${testing_id}",
+        "awslogs-group": "${log_group}"
+      }
+    },
+    "cpu": 1,
+    "mountPoints": [
+        {
+            "readOnly": true,
+            "containerPath": "/rootfs/proc",
+            "sourceVolume": "proc"
+        },
+        {
+          "readOnly": true,
+          "containerPath": "/rootfs/dev",
+          "sourceVolume": "dev"
+        },
+        {
+          "readOnly": true,
+          "containerPath": "/sys/fs/cgroup",
+          "sourceVolume": "al2_cgroup"
+        },
+        {
+          "readOnly": true,
+          "containerPath": "/cgroup",
+          "sourceVolume": "al1_cgroup"
+        },
+        {
+          "readOnly": true,
+          "containerPath": "/rootfs/sys/fs/cgroup",
+          "sourceVolume": "al2_cgroup"
+        },
+        {
+          "readOnly": true,
+          "containerPath": "/rootfs/cgroup",
+          "sourceVolume": "al1_cgroup"
+        }
+    ],
+    "memory": 2048,
+    "portMappings": [
+      {
+          "containerPort": 8125,
+          "protocol": "udp"
+      }
+    ]
+  },
+  {
+      "name": "statsd-client",
+      "image": "alpine/socat:latest",
```

Comment:
I believe socat image is light enough to be used. Also, it would be nice to have something in our test that is used for a public example. https://github.com/aws/amazon-cloudwatch-agent/blob/main/amazon-cloudwatch-container-insights/ecs-task-definition-templates/deployment-mode/sidecar/cwagent-statsd/cwagent-statsd-fargate.json

Reply:
I'm indifferent about netcat vs socat for this